mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -401,10 +401,14 @@ def init_solver(self):
 
     def init_conductors(self):
         print('### Init Diode Conductors Setup ###')
+        # Create source conductors a.k.a the cathode
         self.cathode = assemblies.Cathode(
             V=self.V_CATHODE, T=self.CATHODE_TEMP, WF=self.CATHODE_PHI
         )
 
+        # Create ground plate a.k.a the anode
+        # Subtract small z value for the anode plate not to land exactly on the
+        # grid
```

Comment:
These comments look like a reversion (they're partly unnecessary and partly incorrect)